cura/Machines/Models/QualityManagementModel.py

```diff
@@ -253,27 +292,53 @@ def _update(self):
             return
 
         item_list = []
-        # Create quality group items
+        # Create quality group items (intent category = "default")
         for quality_group in quality_group_dict.values():
             if not quality_group.is_available:
                 continue
 
             item = {"name": quality_group.name,
                     "is_read_only": True,
                     "quality_group": quality_group,
-                    "quality_changes_group": None}
+                    "quality_type": quality_group.quality_type,
+                    "quality_changes_group": None,
+                    "intent_category": "default",
+                    "section_name": catalog.i18nc("@label", "Default"),
+                    }
             item_list.append(item)
         # Sort by quality names
         item_list = sorted(item_list, key = lambda x: x["name"].upper())
 
+        # Create intent items (non-default)
+        available_intent_list = IntentManager.getInstance().getCurrentAvailableIntents()
+        available_intent_list = [i for i in available_intent_list if i[0] != "default"]
+        result = []
+        for intent_category, quality_type in available_intent_list:
+            result.append({
+                "name": quality_group_dict[quality_type].name,  # Use the quality name as the display name
+                "is_read_only": True,
+                "quality_group": quality_group_dict[quality_type],
+                "quality_type": quality_type,
+                "quality_changes_group": None,
+                "intent_category": intent_category,
+                "section_name": IntentCategoryModel.name_translation.get(intent_category, catalog.i18nc("@label", "Unknown")),
```

Comment:
Same as above. You can just directly ask the catalog!

Reply:
Fixed in 2033f24a1